pkgs/applications/window-managers/sway/lock-effects.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub
+, meson, ninja, pkgconfig, scdoc
+, wayland, wayland-protocols, libxkbcommon, cairo, gdk-pixbuf, pam
+}:
+
+stdenv.mkDerivation rec {
+  pname = "swaylock-effects";
+  version = "1.6.0";
+
+  src = fetchFromGitHub {
+    owner = "mortie";
+    repo = "swaylock-effects";
+    rev = "edac98443bffe351a5e095cd625c95c6e8d6bfe5";
+    sha256 = "11nap4xylq48pv7qaqxdcz3lbb2hyki8r996rzfzpw467mkxhsr5";
+  };
+
+  nativeBuildInputs = [ meson ninja pkgconfig scdoc ];
+  buildInputs = [ wayland wayland-protocols libxkbcommon cairo gdk-pixbuf pam ];
+
+  mesonFlags = [
+    "-Dpam=enabled" "-Dgdk-pixbuf=enabled" "-Dman-pages=enabled"
+  ];
+
+  meta = with stdenv.lib; {
+    description = "Screen locker with effects for Wayland";
+    longDescription = ''
+      swaylock-effects is a screen locking utility with built-in screenshots and image manipulation effects for Wayland compositors.
+    '';
+    inherit (src.meta) homepage;
```

I don't know if this is frowned upon in general or not, but I would still prefer to see:
```suggestion
    homepage = "https://github.com/mortie/swaylock-effects";
```